### DISCUSSION TITLE
Failed to load rules state error

### DISCUSSION THREAD

**Discussion by AlHood77** [GoogleCloudPlatform/prometheus-engine#303] [Q&A] [answered]
I have just set up a fully managed prometheus service. Also deployed Grafana and the Prometheus-UI and the data source is working as expected. I was adding some Grafana alerts and noticed this error message on the Grafana alerting UI.

I'm not sure what it means. It is not affecting anything I'm doing at the moment. Just trying to understand why it's happening?

![Screenshot 2022-08-11 at 10 37 49](https://user-images.githubusercontent.com/45075866/184106346-5213ba53-365c-4523-ac4e-5b7f874b87f5.png)
.

**Comment by lyanco** [comment, marked as answer]
The /rules endpoint is not implemented (as rules don't run in the same place that queries run) - see https://github.com/GoogleCloudPlatform/prometheus-engine/issues/288

The UI is likely trying to give you a unified rules view across your Grafana alerts and Prometheus alerts, but the /rules endpoint is 404ing, giving you that error.

**Comment by pintohutch** [comment]
Somewhat related is https://github.com/GoogleCloudPlatform/prometheus-engine/issues/291, which is in our backlog.
